hatice-ecevit-2016400138/assignment1/Anagram.java

```diff
@@ -63,6 +63,8 @@ public static boolean areAnagrams(String s1, String s2, boolean caseSensitive){
     public static boolean areAnagramSentences(String s1,String s2, boolean caseSensitive){
         String[] words1 = s1.split("\\s+");
         String[] words2 = s2.split("\\s+");
+        Boolean[] isMatched1 = new Boolean[words1.length];
```

Comment:
Optional: you define `anagram1` and `anagram2` below, so it makes sense to do the same for these (since they are not used before then.